client/src/app/validators/email-correct.validator.ts

```diff
@@ -0,0 +1,16 @@
+import { FormControl } from '@angular/forms';
+
+export class EmailValidator {
+  static emailValidator(control: FormControl): { [key: string]: boolean} {
+    // tslint:disable-next-line: max-line-length
+    const emailRegex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
is there a way to format this and add line break somewhere and don't use tslint disable?